packages/help-center/src/hooks/use-opening-coordinates.ts

```diff
@@ -31,8 +44,17 @@ export const calculateOpeningPosition = ( element: HTMLElement ) => {
 
 	const { x, y, width, height } = element.getBoundingClientRect();
 
-	const buttonLeftEdge = x;
-	const buttonRightEdge = x + width;
+	const position = getPosition( element );
+
+	// handle RTL languages
+	const buttonLeftEdge = x - position;
+
+	const buttonRightEdge =
+		x +
+		width +
+		( element.classList.contains( 'masterbar__item' )
```

Comment:
Would it make more sense to put this below where we actually calculate the coordinates? This small padding should not cause it to go off screen anywhere and at least it would keep all the adjustments in one place to better follow.

```
		coords.left = buttonRightEdge + offestToAlignWithIcon - helpCenterWidth;
```

Reply:
Also, any reason you don't use `position` here again?

Reply:
Have just pushed it :)